Update ecovacs.markdown clean rooms, remote control robot (#42590)
home-assistant/home-assistant.github.io: source/_integrations/ecovacs.markdown

{% enddetails %}

### useful commands
#### clean specific room(s)

> You can command the robot to clean specific rooms by using the `vacuum.send_command` service, the value parameter should be a string where the first number is the clean count and the second the room number (can be found as attribute of the vacuum entity) you can list multiple rooms by separating them with an `;`

Reviewer: ```suggestion
You can command the robot to clean specific rooms by using the `vacuum.send_command` service, the `value` parameter should be a string where the first number is the clean count and the second the room number. These details can be found as state attributes of the vacuum entity. You can list multiple rooms by separating them with an `;`.
```

```yaml
data:

>   command: clean_V2

Reviewer: All the other parameters need an explanation too, not just the `value` parameter.

  params:
    act: start
    content:
      type: freeClean

>       value: 1,3;1,7

Reviewer: ```suggestion
      value: "1,3;1,7"
```

> action: vacuum.send_command

Reviewer: Put the action at the top.

target:
  device_id: 058a9bf714d680fdd9e69a7c292fc39d
```
#### go to point on the map

You can request the robot to go to a specific point on the map. Get the coordinates with `raw_get_positions`

```yaml
action: vacuum.send_command

> metadata: {}

Reviewer: Not relevant.

```suggestion
```

> target:

Reviewer: Please keep the order of the data parameters consistent. If we end with the target parameter in the example above, we should do that here too.

  device_id: 058a9bf714d680fdd9e69a7c292fc39d
data:
  command: clean_V2
  params:
    act: start
    content:
      type: mapPoint
      value: "-2900,-5500"
```

#### remote control

move robot (`act` can be either `forward|backward|turnLeft|turnRight|stop`)

```yaml
action: vacuum.send_command
target:
  device_id: 058a9bf714d680fdd9e69a7c292fc39d
data:
  command: move
  params:
    act: forward

>     a: 0

Reviewer: With does the `a` parameter mean?
